src/conn/conn_capacity.c

```diff
@@ -81,7 +101,8 @@ __capacity_config(WT_SESSION_IMPL *session, const char *cfg[])
          */
         cap->threshold = ((cap->ckpt + cap->evict + cap->log) / 100) * WT_CAPACITY_PCT;
         if (cap->threshold < WT_CAPACITY_MIN_THRESHOLD)
-            cap->threshold = WT_CAPACITY_MIN_THRESHOLD;
+            cap->threshold = WT_MIN(WT_CAPACITY_MIN_THRESHOLD, cap->total);
```

Comment:
Could you explain the logic of this change? I don't completely get it.

Reply:
mainly to avoid a long time without triggering fsyn. after we add fsync_background_period_sec Feature, there is no need to change here.

Reply:
Cool - thanks for explaining that.